stackdriver/README.md

```diff
@@ -0,0 +1,61 @@
+# stackdriver
+
+This gem is a convenience package for loading all Stackdriver gems in the google-cloud-ruby project. 
+- [google-cloud-logging](../google-cloud-logging)
+- [google-cloud-error_reporting](../google-cloud-error_reporting)
+- [google-cloud-monitoring](../google-cloud-monitoring)
+
+Please see the top-level project [README](../README.md) for more information about the individual Stackdriver google-cloud-ruby gems.
+
+## Quick Start
+
+```sh
+$ gem install stackdriver
+```
+
+## Overview
```

Comment:
This approach is a bit confusing. Some users may think they need to include the individual require statements even when the stackdriver gem is present. Instead, just show them what they need to do (require "stackdriver"), describe what it does, and point them to where they can get more detailed information.